src/Search/Microsoft.Azure.Search/Customizations/Indexers/Models/IndexingParametersExtensions.cs

```diff
@@ -13,6 +13,22 @@ namespace Microsoft.Azure.Search.Models
     /// </summary>
     public static class IndexingParametersExtensions
     {
+        /// <summary> 
+        /// Specifies that the indexer will index only the storage metadata and completely skip the document extraction process. This is useful when 
+        /// you don't need the document content, nor do you need any of the content type-specific metadata properties. 
+        /// See <see href="https://azure.microsoft.com/en-us/documentation/articles/search-howto-indexing-azure-blob-storage/" /> for details. 
```

Comment:
@seansaleh Good catch. It looks like we missed this one because the code was removed and then re-added in a subsequent commit.